src/api/arbo-typedefs.ts

```diff
@@ -30,6 +31,7 @@ export const arboTypedefs = `
     sex: String
     sourceSheetId: String
     sourceSheetName: String
+    unRegion: String
```

Comment:
Optional because of Kosovo + Taiwan. On our country list but not categorized by the UN.